go.mod

```diff
@@ -88,12 +88,12 @@ require (
 	github.com/prometheus/common v0.10.0
 	github.com/rancher/apiserver v0.0.0-20210209001659-a17289640582
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
-	github.com/rancher/eks-operator v1.0.6
-	github.com/rancher/gke-operator v1.0.2-rc1
+	github.com/rancher/eks-operator v1.1.1-rc3
```

Comment:
Is EKS operator on the same version as gke ? I thought it was still on 1.0.x for 2.5

Reply:
Will have to backport in order to keep the 1.0.x version https://github.com/rancher/eks-operator/pull/43

Reply:
Yes we decided that EKS operator should stay at 1.0.x for 2.5.x and move to 2.0.0 for 2.6

Reply:
I see that we wen't with 1.1.1 for 2.6 at first, that will change to 2.0.0 and will be tagged of master. So any fixes that need to go into 2.5 should be backported to 1.0.x